test/e2e/specs/specs-playwright/wp-blocks__coblocks.ts

```diff
@@ -87,12 +91,15 @@ describe( DataHelper.createSuiteTitle( 'Blocks: CoBlocks' ), () => {
 		${ DynamicHRBlock }    | ${ null }
 		${ HeroBlock }         | ${ [ heroBlockHeading ] }
 		${ ClicktoTweetBlock } | ${ [ clicktoTweetBlockTweet ] }
-		${ LogosBlock }        | ${ [ path.parse( logoImage ).name ] }
 	`(
 		`Confirm $block.blockName block is visible in published post`,
 		async ( { block, content } ) => {
 			// Pass the Block object class here then call the static method to validate.
 			await block.validatePublishedContent( page, content );
 		}
 	);
+
+	it( `Confirm Logso block is visible in published post`, async () => {
```

Comment:
A side effect of the main change (move `createTestImage` inside a hook) is that we can't use `logoImage` to define a parameter anymore, as it doesn't exist when the test is _defined_.

Reply:
That's a bummer, but small price to pay for this change.